packages/blade/.storybook/react/global.css

```diff
@@ -2,3 +2,14 @@
   /* Need this to ensure mdx stories don't break visually */
   box-sizing: border-box;
 }
+
+body h1,
```

Comment:
only h1 should be enough right? it applies to h1 globally?

Reply:
nope all headings use Tasa Orbiter no. So changed it for all hx tags. This is for Storybook's own styling btw.